sycl/include/CL/sycl/detail/property_helper.hpp

```diff
@@ -85,5 +84,5 @@ template <int ID> class PropertyWithData : public PropertyWithDataBase {
 
 } // namespace detail
 
-} // namespace sycl
-} // __SYCL_INLINE_NAMESPACE(cl)
+} // __SYCL_OPEN_NS()
+__SYCL_CLOSE_NS()
```

Comment:
The comment seems to be unnecessary:
```suggestion
} __SYCL_CLOSE_NS()
```
I think this notation makes it clear that this bracket closes SYCL namespace.

Reply:
This is how clang format thinks it should look like.